website/docs/configuration/snippet-choice.md

```diff
@@ -0,0 +1,192 @@
+# The snippet choice feature
+
+## Introduction
+
+Snippets are short pieces of code. They might come from difference public sources such as GitHub, GitLab, Stackoverflow,
+ etc.
+
+So-called snippet scanners like ScanOSS and FossID scrape those public sources and build a knowledge base of snippets,
+classifying them by their author, version and license. Then, the snippet scanner component of such products can use this
+knowledge base to find if some source file contains some of those snippets. The matching of snippet can be:
+
+* *full*: the whole snippet matches the source file
+* *partial*: only some parts of a snippet matches the source file
+
+Currently, ORT supports two products offering snippet scanners capabilities: ScanOSS and FossID. While each
+implementation is specific, the base workings are the same: ORT submits a source file to scan to the snippet scanner and
+receives the list of the snippets matching some parts of this source file. ORT puts those snippets in the
+`SnippetFindings` property of the [ScanSummary](https://github.com/oss-review-toolkit/ort/blob/main/model/src/main/kotlin/ScanSummary.kt).
+
+## The problem
+
+As mentioned in the previous section, ORT returns the list of snippets (i.e. matches) for a given source file. These
+snippets are aggregated by source code location, including the line ranges.
+
+ORT lacks a mechanism to control the snippets results: while some of these results are legitimate, others are false
+positives. The snippet choice feature aims at filling this gap by allowing the user to:
+
+* Choose a snippet as the origin of the code snippet found in the project source code, discarding all other snippets for
+this source location.
+* Mark all the snippets for a given source location as false positives.
+
+Hence, the user can control which snippets are present in the ORT snippet results.
+
+## Choosing a snippet
+
+Let's say a source file `ci.yml` has been scanned with the following findings:
+
+```yaml
```

Comment:
Please use only two spaces for indentation in YAML snippets, and align list entries to directly below the parent node, to match our usual serialization format. Like:

```
snippets:
- source_location:
    path: ".github/workflows/ci.yml"
    start_line: 3
    end_line: 17
  snippets:
    ...
```
```